src/plugins/intel_cpu/src/nodes/reference.cpp

```diff
@@ -123,8 +123,9 @@ ov::TensorVector Reference::prepareInputs() const {
     ov::TensorVector inputs;
     for (size_t i = 0; i < inputShapes.size(); i++) {
         void *srcDataPtr = getParentEdgesAtPort(i)[0]->getMemory().getData();
-        inputs.push_back(ov::Tensor(ngraphOp->get_input_element_type(i),
-                                             getParentEdgesAtPort(i)[0]->getMemory().getStaticDims(), srcDataPtr));
+        ov::Shape shape = ngraphOp->get_input_partial_shape(i).rank().get_length() == 0 ?
+                ov::Shape{} : getParentEdgesAtPort(i)[0]->getMemory().getStaticDims();
+        inputs.push_back(ov::Tensor(ngraphOp->get_input_element_type(i), shape, srcDataPtr));
```

Comment:
What about the output tensors? Most likely we have to modify their shapes as well. Also please consider the case when the output tensors shapes aren't known in advance (internal dynamism).

Reply:
Done